b2b-sample/libs/business-admin-app/data-access/data-access-common-models-util/src/lib/identityProvider/identityProviderUtils.ts

```diff
@@ -17,10 +17,30 @@
  */
 
 import { getManagementAPIServerBaseUrl, getOrgUrl } from "@b2bsample/shared/util/util-application-config-util";
-import { ENTERPRISE_ID, GOOGLE_ID } from "@b2bsample/shared/util/util-common";
+import { EMPTY_STRING, ENTERPRISE_ID, GOOGLE_ID } from "@b2bsample/shared/util/util-common";
 import IdentityProviderDiscoveryUrl from "./identityProviderDiscoveryUrl";
 import IdentityProviderTemplateModel from "./identityProviderTemplateModel";
-import config from "../../../../../../../config.json";
+import enterpriseImage from "../../../../../ui/ui-assets/src/lib/images/enterprise.svg";
+import googleImage from "../../../../../ui/ui-assets/src/lib/images/google.svg";
+
+/**
+ * @param templateId - template id of the identity provider
```

Comment:
add a new line here to make it consistent.

Reply:
Fixed with https://github.com/wso2/samples-is/pull/416/commits/e238027e3ab758576ae189fd6ec3d1496abb6315